cl/search/documents.py

```diff
@@ -539,3 +547,276 @@ def prepare_person_child(self, instance):
 
     def prepare_school(self, instance):
         return [e.school.name for e in instance.educations.all()] or None
+
+
+# RECAP
+class DocketBaseDocument(Document):
+    docket_child = JoinField(relations={"docket": ["recap_document"]})
+    timestamp = fields.DateField()
+
+    class Django:
+        model = Docket
+        ignore_signals = True
+
+    def prepare_timestamp(self, instance):
+        return datetime.utcnow()
+
+
+@recap_index.document
+class RECAPDocument(DocketBaseDocument):
+    id = fields.IntegerField(attr="pk")
+    docket_entry_id = fields.IntegerField(attr="docket_entry.pk")
+    description = fields.TextField(
+        attr="docket_entry.description",
+        analyzer="text_en_splitting_cl",
+        fields={
+            "exact": fields.TextField(
+                attr="docket_entry.description", analyzer="english_exact"
+            ),
+        },
+        search_analyzer="search_analyzer",
+    )
+    entry_number = fields.IntegerField(attr="docket_entry.entry_number")
+    entry_date_filed = fields.DateField(attr="docket_entry.date_filed")
+    entry_date_filed_text = fields.TextField(
+        analyzer="text_en_splitting_cl",
+        fields={
+            "exact": fields.TextField(analyzer="english_exact"),
+        },
+        search_analyzer="search_analyzer",
+    )
```

Comment:
I don't think we need this one. It was just so people could search for Sept. and get back things in September. For RECAP, I don't think that's really useful.

Reply:
Got it, I've removed this field.